### ISSUE TITLE
Protect 1.18.0 seems to have broken streams

### ISSUE THREAD

**Issue by jasonlboggs** [hjdhjd/homebridge-unifi-protect#390]
**Describe The Problem:**
After upgrading UNVR to 1.18.0, the streams no longer load in the Home app.  Thumbnails still load.

**To Reproduce:**
Ensure UNVR is on 1.18.0
Attempt to open a stream for any camera (I'm trying on a G3 Dome and G3 Instant)

**Logs:**
[5/13/2021, 8:20:46 AM] [UniFi Protect] UNVR [UNVR4] Back Yard [UVC G3 Dome]: Streaming request from 10.xx.xx.xx: 1280x720@30fps, 299 kbps. Using RTSP stream profile: 1920x1080@30fps (High), 6000 kbps.

**Homebridge Configuration:**
"platform": "UniFi Protect",
            "controllers": [
                {
                    "address": "10.xx.xx.xx",
                    "password": "redacted",
                    "username": "homebridge",
                    "motionDuration": 30,
                    "options": [
                        "Disable.Motion.Switch",
                        "Disable.Motion.Trigger"
                    ]
                }
            ]

**Screenshots:**
N/A

**Environment:**

* **Homebridge Version:** 1.3.4
* **Node Version:** 14.16.1
* **Homebridge-UniFi-Protect Plugin Version:** 
* **UniFi Protect Version:** 1.18.0
* **UniFi Protect Controller Type:** UNVR
* **UniFi Protect Controller Firmware Version:** 1.3.39
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** 11 Pro Max - 14.5.1
* **Operating System and OS Version:** Ubuntu Server 20.04.2

**Other info**:
In the Protect release notes for 1.18.0 (https://community.ui.com/releases/UniFi-Protect-Application-1-18-0/74993532-682a-4262-88d4-cb6c1b91637e) they mention **Cameras: faster Live stream startup time.** This may have something to do with the issue.

Thank you so much for providing this plugin!

**Comment by wingcomm** [comment]
I'm seeing the same issue as well.

**Comment by hjdhjd** [comment]
Thanks for the support, really appreciate it @jasonlboggs. 

I’ve got a similar setup, on v1.18.0 and don’t have these issues. You might want to try the newly production 1.18.1 which just came out and see if it helps. I don’t think this is a plugin issue. If you’re still seeing issues, try looking at the troubleshooting documentation for the plugin and see if any of that helps in your situation.

**Comment by jasonlboggs** [comment]
Thanks for encouraging me to look at the troubleshooting steps. I discovered that my NIC was still using a secondary IP that I used a while back while migrating my DNS/DHCP server. Once I forcefully removed the IP address and rebooted Homebridge, everything was working properly. 

@wingcomm  make sure you don’t have any secondary IP addresses in ‘ip addr’ and make sure the Homebridge status page shows the correct IP address.
